beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -1319,8 +1320,10 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         block: SignedBeaconBlock<T::EthSpec>,
     ) -> Result<GossipVerifiedBlock<T>, BlockError<T::EthSpec>> {
         let slot = block.message.slot;
-        let graffiti_string = String::from_utf8(block.message.body.graffiti[..].to_vec())
-            .unwrap_or_else(|_| format!("{:?}", &block.message.body.graffiti[..]));
+        let re = Regex::new("\\p{C}").expect("regex is valid");
```

Comment:
```suggestion
        #[allow(clippy::invalid_regex)]
        let re = Regex::new("\\p{C}").expect("regex is valid");
```

Reply:
It seems I was wrong about the raw string being part of the problem. Clippy is really wrong here, so I think we should just bypass it.

Reply:
Frustrating, appreciate the review!